src/schema/core.yaml

```diff
@@ -1340,26 +1346,21 @@ classes:
     description: >-
       An atom or molecule that can be represented with a chemical formula. Include lipids, glycans, natural products, drugs.
       There may be different terms for distinct acid-base forms, protonation states
-    comments:
-      - As with the parent OntologyClass, we will not assign an nmdc id pattern or typecode to this class.
     slots:
       - alternative_names
       - chemical_formula
       - inchi
       - inchi_key
       - smiles
+    slot_usage:
+      id:
+        required: true
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:chem-{id_shoulder}-{id_blade}$"
```

Comment:
this may need to change to a ChEBI prefix pattern per our discussions with Chris